app/models/manageiq/providers/kubevirt/infra_manager/vm/operations/infra_volume.rb

```diff
@@ -0,0 +1,157 @@
+module ManageIQ::Providers::Kubevirt::InfraManager::Vm::Operations::InfraVolume
+  def raw_attach_volume(vm, pvc_name, volume_name = nil)
```

Comment:
You have `vm` as a param here but this is an instance method in the vm class so you can just use `self` for anything that you need, e.g. instead of `ems = vm.ext_management_system` you can just use `ext_management_system` and that is effectively `self.ext_management_system`